webapp/pages/post/_id/_slug/index.vue

```diff
@@ -125,27 +125,78 @@
                 @toggleObservePost="toggleObservePost"
               />
             </div>
-            <!-- Comments -->
+            <!-- comments -->
             <ds-section>
+              <!-- comment list -->
               <comment-list
                 :post="post"
                 @toggleNewCommentForm="toggleNewCommentForm"
                 @reply="reply"
               />
               <ds-space margin-bottom="large" />
+              <!-- commenting form -->
               <comment-form
                 v-if="showNewCommentForm && !isBlocked && canCommentPost"
                 ref="commentForm"
                 :post="post"
                 @createComment="createComment"
               />
-              <ds-placeholder v-if="isBlocked">
-                {{ $t('settings.blocked-users.explanation.commenting-disabled') }}
-                <br />
-                {{ $t('settings.blocked-users.explanation.commenting-explanation') }}
-                <page-params-link :pageParams="links.FAQ">
-                  {{ $t('site.faq') }}
-                </page-params-link>
+              <!-- commenting impossible -->
+              <ds-placeholder v-else>
+                <hc-empty
+                  margin="xxx-small"
+                  icon="messages"
+                  :message="$t('settings.blocked-users.explanation.commenting-disabled')"
+                />
+                <!-- blocked author -->
+                <ds-space v-if="isBlocked" centered margin="xxx-small">
+                  <ds-space margin-bottom="small" />
+                  <ds-heading tag="h4">
+                    {{ $t('contribution.comment.commenting-disabled.blocked-author.reason') }}
+                  </ds-heading>
+                  <ds-text>
+                    {{
+                      $t('contribution.comment.commenting-disabled.blocked-author.call-to-action')
+                    }}
+                  </ds-text>
+                  <nuxt-link :to="authorLink">
+                    <base-button icon="arrow-right" filled>
+                      {{
+                        $t('contribution.comment.commenting-disabled.blocked-author.button-label', {
+                          name: post.author.name,
+                        })
+                      }}
+                    </base-button>
+                  </nuxt-link>
+                </ds-space>
+                <!-- no group member -->
+                <ds-space v-else-if="!canCommentPost" centered margin="xxx-small">
```

Comment:
`canCommentPost` seems a bad naming to me because it's not specified enough but to general.